test/shard.test.js

```diff
@@ -12,14 +11,13 @@ const Schema = mongoose.Schema;
 const uri = process.env.MONGOOSE_SHARD_TEST_URI;
 
 if (!uri) {
-  console.log(
-    chalk.red(
-      '\n', 'You\'re not testing shards!',
-      '\n', 'Please set the MONGOOSE_SHARD_TEST_URI env variable.', '\n',
-      'e.g: `mongodb://localhost:27017/database', '\n',
-      'Sharding must already be enabled on your database'
-    )
-  );
+  console.log([
+    '\x1b[31m',
```

Comment:
Nitpick: I'd add `const redColorEscapeCharacter = '\x1b[31m';` and use it in the console.log to reveal intent.

Reply:
done